velox/expression/Expr.cpp

```diff
@@ -509,6 +509,29 @@ void setDictionaryWrapping(
 }
 } // namespace
 
+void setConstantWrappingAndTranslateRows(
```

Comment:
Yes, I agree and your proposal sounds good to me. I'll rename the added functions in this diff, and we can rename existing functions in a separate diff. Does this sound good?

Reply:
I find the names and meaning of these function difficult to understand. These functions are quite small. I wonder if it make sense to just write out that code explicitly in the caller(s)?

> setDictionaryWrapping or setConstantWrapping sound like they are applying wrapping, whereas they are storing the state of a successful peeled operation. How about we use the term 'peel' instead of wrap to signify that we are storing a peeled wrap which then needs to be applied back.
eg setConstantPeelAndPeelRows , setDictionaryPeel
similarly also rename the functions and member variables in EvalCtx.

The naming can be improved indeed. Consider sending a PR to propose better names.

Reply:
I created these functions because these logics are used in both Expr.cpp and CastExpr.cpp. I feel extracting and reusing them would make it easier to maintain the code in the future.

Reply:
Inlined these functions in the callers according to the offline discussion.